packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -474,6 +474,9 @@ abstract class WidgetsBinding extends BindingBase with SchedulerBinding, Gesture
   int _deferFirstFrameReportCount = 0;
   bool get _reportFirstFrame => _deferFirstFrameReportCount == 0;
 
+  /// Whether the first frame has finished rendering.
+  bool get debugDidSendFirstFrameEvent => !_needToReportFirstFrame;
```

Comment:
These docs should mention:
- that this is only valid in debug and profile builds, it can't be used in release builds
- that it can be deferred using [deferFirstFrameReport] and [allowFirstFrameReport]
- that it is set at the end of the call to drawFrame

Reply:
Done.